app/views/accessibility/audits/index.html

```diff
@@ -11,8 +11,8 @@
   <div class="govuk-warning-text">
     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
     <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-    Services built using the prototype kit, must not be presented for audits. They are not intended for production quality products or services.
+      <span class="govuk-warning-text__assistive"> Warning </span>
```

Comment:
Can you explain why you've made these changes to the warning component and put the text in quotes.

Reply:
@andyjones81 the warning component is currently not displaying correctly on the page:

<img width="632" alt="image" src="https://github.com/DFE-Digital/design/assets/97228674/bd4e59e2-fb76-4635-967e-23d5daa672c6">

This was my attempt at fixing it 🙂